lib/khttp/kclient/BUILD.bazel

```diff
@@ -1,9 +1,15 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
 go_library(
-    name = "go_default_library",
+    name = "kclient",
     srcs = ["client.go"],
     importpath = "github.com/enfabrica/enkit/lib/khttp/kclient",
     visibility = ["//visibility:public"],
-    deps = ["//lib/kflags:go_default_library"],
+    deps = ["//lib/kflags"],
+)
+
+alias(
+    name = "go_default_library",
+    actual = ":kclient",
+    visibility = ["//visibility:public"],
 )
```

Comment:
I assume you're adding these aliases because to not update them would require many more file changes, and this keeps the already-huge PR line diff smaller.

i.e. Are these added only in the places they are needed to pass `bazel build` / `bazel test`?

Reply:
Since internal imports this repository, I'm nervous about a change here that breaks internal mostly.

They're currently everywhere - gazelle gets put in a mode where it creates both (which is the simplest to manage).

After we're sure internal can handle it, I can send a similar change that puts gazelle in the new-only mode, deleting all the aliases

Reply:
sgtm